config/webhook-client.php

```diff
@@ -68,4 +68,9 @@
      * It deletes all records after 1 week. Set to null if no models should be deleted.
      */
     'delete_after_days' => 30,
+
+    /*
+     * Should a unique token be added to the route name
+     */
+    'add_unique_token_to_route_name' => false,
```

Comment:
@it-can Just personal preference but I would probably simplify this to `'tokenize_route_name'`.

Reply:
i think freek should decide on this one

Reply:
Although `tokenize_route_name` would be good I think`add_unique_token_to_route_name` is more clear / human.